package.json

```diff
@@ -198,4 +198,4 @@
     "vinyl-sourcemaps-apply": "0.2.1",
     "watchify": "3.11.1"
   }
-}
+}
```

Comment:
I think this change was accidental. You can revert with `git checkout master package.json`

Reply:
Added regenerator-runtime to use await keyword.